### PR TITLE
op-tee patch set: Add keymaster and gatekeeper

### PR DESCRIPTION
**Pull request by rshym**

### CODE REVIEW

File: device.mk [hunk 1]
```diff
@@ -150,9 +150,24 @@ PRODUCT_PACKAGES += \
     android.hardware.health@1.0-impl \
     android.hardware.health@1.0-service
 
-# Keymaster HAL
+# Security
```
Comment (arminn) [inline device.mk]: Maybe it is a good point to move all security into security.mk
And guard all non generic dependencies(like OPTEE_PLATFORM_FLAVOR, OPTEE_PLATFORM and *.ta ??) with TARGET_BOARD_PLATFORM.

File: ueventd.xenvm.rc [hunk 2]
```diff
@@ -3,3 +3,5 @@
 /dev/pvr_sync		0666	system		graphics
 /dev/dri/controlD64	0660	system		graphics
 /dev/ttyUSB0		0666	system		system
+/dev/teepriv0		0660	system		keystore
+/dev/tee0			0660	system		keystore
```
Comment (arminn) [inline ueventd.xenvm.rc]: whitespace count?

File: device.mk (hunk 1 above)
Comment (rshym) [reply device.mk]: Done.

File: ueventd.xenvm.rc (hunk 2 above)
Comment (rshym) [reply ueventd.xenvm.rc]: Fixed.

**Pull request merged**